libact/query_strategies/multiclass/tests/test_hierarchical_sampling.py

```diff
@@ -0,0 +1,52 @@
+""" UncertaintySampling test
+"""
+import unittest
+
+import numpy as np
+from sklearn import datasets
+from sklearn.utils import shuffle
```

Comment:
You may have to mock `sklearn.utils` here so the documentation building on RTD won't fail.
https://github.com/ntucllab/libact/blob/master/docs/conf.py#L27